### ISSUE TITLE
[Language Selector UI]: Dropdown cannot be opened by keyboard

### ISSUE THREAD

**Issue by techtolentino** [Qiskit/platypus#1506]
Originally discovered/reported by @eddybrando 

### Steps to reproduce the problem

1. Go to https://learn.qiskit.org/course/introduction/why-quantum-computing
2. Keyboard tab your way through the UI until you reach the language selector
4. See that using the `enter` or `space` key does not open the dropdown

### What is the current behavior?

Using the `enter` or `space` key does not trigger/open the dropdown

### What is the expected behavior?

Using the `enter` or `space` key should trigger/open the dropdown

### Screenshots

https://user-images.githubusercontent.com/6276074/187951386-2f4f7f21-7d12-4c0d-97c3-f721eb5c5f23.mov

### Additional information

The underlying component is the [bx-dropdown](https://web-components.carbondesignsystem.com/?path=/story/components-dropdown--default)
